bfcache/http_server.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Since this test app isn't a PWA, we can omit this python file and update README to use `python3 -m http.server 8000` instead.

Reply:
Done, thanks!